nengo_gui/static/components/netgraph.js

```diff
@@ -307,9 +307,6 @@ Nengo.NetGraph.prototype.on_message = function(event) {
         var item = this.svg_objects[data.uid];
         item.set_label(data.name);
 
-        var item_mini = this.minimap_objects[data.uid];
-        item_mini.set_label(data.name);
-
```

Comment:
ahhh yeah i think that might just be a holdover from me copying all the svg_objects code for the minimap when i was getting it started, i'm pretty sure i never used the label for anything